packages/compile-vyper/index.js

```diff
@@ -92,7 +92,7 @@ function execVyper(options, source_path, callback) {
   ) {
     formats.push("source_map");
   }
-  const command = `vyper -f${formats.join(",")} ${source_path}`;
+  const command = `vyper -f ${formats.join(",")} ${source_path}`;
```

Comment:
Can you explain why you made this change here?

Reply:
Because the vyper tests are failing and I wanted to see if something changed that is suddenly making this fail. I'm having trouble upgrading the vyper compiler and I happened to notice this lack of a space between the option flag and the arguments. I want to see if this will make the tests pass.

Reply:
No dice, but there should be a space here anyways according to the docs. https://vyper.readthedocs.io/en/stable/compiling-a-contract.html